product_register.php

```diff
@@ -2,32 +2,23 @@
 ini_set('display_errors', 1);
 ini_set('display_startup_errors', 1);
 error_reporting(E_ALL);
-ini_set('upload_max_filesize', '2M');
+
 // Include the constant file
 require_once 'helper/validation.php';
 require_once 'config/constants.php';
+require_once 'libraries/db.php';
```

Comment:
require_once 'config/constants.php';

No need to include the file again. As you already including this file in db.php